packages/trpc/server/routers/viewer/teams/search.handler.ts

```diff
@@ -0,0 +1,93 @@
+import type { PrismaClient } from "@prisma/client";
+
+import { WEBAPP_URL } from "@calcom/lib/constants";
+import type { TrpcSessionUser } from "@calcom/trpc/server/trpc";
+
+import type { TSearchInputSchema } from "./search.schema";
+
+type SearchHandlerOptions = {
+  ctx: {
+    user: NonNullable<TrpcSessionUser>;
+    prisma: PrismaClient;
+  };
+  input: TSearchInputSchema;
+};
+
+export async function searchHandler({ input, ctx }: SearchHandlerOptions) {
+  const { cursor, limit: limit_, teamId, search } = input;
+  const limit = limit_ ?? 10;
+
+  const items = await ctx.prisma.membership.findMany({
+    take: limit + 1,
+    select: {
+      accepted: true,
+      role: true,
+      disableImpersonation: true,
+      user: {
+        select: {
+          username: true,
+          email: true,
+          name: true,
+          id: true,
+          bio: true,
+        },
+      },
+      teamId: true,
+      userId: true,
+    },
+    where: {
+      teamId,
+      ...(search &&
+        search !== "" && {
```

Comment:
isn't `search !== ""` redundant ?
`""` is a falsy value
https://developer.mozilla.org/en-US/docs/Glossary/Falsy